src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1006,6 +1006,45 @@ void Method()
 //]
     }
 }
+";
+
+        private const string s_preferExpressionBodyForLocalFunctions = @"
+using System;
+
+//[
+class Customer
+{
+    private int Age;
+
+    public int GetAge() 
+    {
+        return GetAgeLocal();
+        
+        int GetAgeLocal() => this.Age;
```

Comment:
>int GetAgeLocal() => this.Age; [](start = 8, length = 30)

nit: we recommend camelCasing for local functions and locals.

Reply:
Are we using camel case for local function in roslyn?

Reply:
I just heard that IDE uses Pascal for local functions. Compiler users camel.